plutus-conformance/README.md

```diff
@@ -0,0 +1,83 @@
+# Plutus Core Conformance Test Suite
+
+## Specification
```

Comment:
wrong content

Reply:
Let's at least have a basic readme, and probably we should attribute the test sources immediately.